### PR TITLE
ansible-galaxy: add role info and collection info examples

### PR DESCRIPTION
**Pull request by papanito**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by schneiderl — APPROVED** [review]

File: pages/common/ansible-galaxy.md [hunk 1]
```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the role id for a role name `YourUser.RoleName`:
+
+`ansible-galaxy info YourUser.RoleName | grep -E 'id: [0-9]' | awk {'print $2'}`
```
[the remark this answers is not in the document]
Comment (papanito) [reply pages/common/ansible-galaxy.md:28]: Yes... I will make that more clear

File: pages/common/gpg-keyserver.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+> See `gpg2` for GNU Privacy Guard 2.
+> More information: <https://gnupg.org>.
+
+- Encrypt a directory into `archive.gpg` using a passphrase:
+
+`gpg --keyserver pool.sks-keyservers.net --recv-key {{fingerprint}}`
+
+- Decrypt `archive.gpg` into a directory of the same name:
+
+`gpg-zip --decrypt {{path/to/archive.gpg}}`
```
Comment (navarroaxel) [inline pages/common/gpg-keyserver.md:13]: Please use another branch *don't close this PR yet*, but `gpg-zip` should be a page with just `gpg-zip` examples, not merged with `gpg --keyserver` that should be in another PR too.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/ansible-galaxy.md [hunk 3]
```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the role id for a role name `{{username.role_name}}`:
```
Comment (marchersimon) [inline pages/common/ansible-galaxy.md:26]: I don't think token syntax in description lines is supported by most clients. How about just
```suggestion
- Get the role id for a role name:
```

File: pages/common/ansible-galaxy.md [hunk 4]
```diff
@@ -22,3 +22,7 @@
...
+
+`ansible-galaxy info {{username.role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```
Comment (marchersimon) [inline pages/common/ansible-galaxy.md:28]:
```suggestion
`ansible-galaxy info {{username}}.{{role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```
Actually, since the dot is mandatory it doesn't go in token syntax

File: pages/common/gpg-keyserver.md (hunk 2 above)
Comment (papanito) [reply pages/common/gpg-keyserver.md:13]: Sorry that change should not be in this pr 🤦

**Comment by marchersimon** [comment]
@papanito please try to avoid force-pushing, since it makes the review process harder and we do a `sqash and merge` anyways.

File: pages/common/ansible-galaxy.md [hunk 5]
```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the role id for a role name:
+
+`ansible-galaxy info {{username}}.{{role_name}} | grep -E 'id: [0-9]' | awk {'print $2'}`
```
Comment (marchersimon) [inline pages/common/ansible-galaxy.md:28]: I'm not really sure if we should include the `grep` and `awk` examples here. In general the `ansible-galaxy` page should show how the command works. If someone wants to filter the role id they can use any tools they want.

Comment (sbrl) [reply pages/common/ansible-galaxy.md:28]: I agree with @marchersimon on this one. we generally want to avoid demonstrating the functionality of other commands here.

Comment (navarroaxel) [reply pages/common/ansible-galaxy.md:28]: @papanito you can remove the `grep` and `awk` parts and explain what the `info` subcommand displays here.
```suggestion
`ansible-galaxy info {{username}}.{{role_name}}
```
Also, if the `.` is mandatory I think we should use the same token syntax with the first 2 examples.

Comment (marchersimon) [reply pages/common/ansible-galaxy.md:28]: @navarroaxel I suggested that above, I didn't see that the first two examples had it in the token syntax.
But why should it not be outside? We're doing that in all pages where two arguments are separated by a single character.

Comment (navarroaxel) [reply pages/common/ansible-galaxy.md:28]: @marchersimon It's like the `ssh.md`: the `@` is outside `{{username}}@{{host}}` because it's mandatory and the `@` is not a placeholder, the same apply here.

Comment (papanito) [reply pages/common/ansible-galaxy.md]: I assume then the first two examples should also be updated?

File: pages/common/ansible-galaxy.md [hunk 6]
```diff
@@ -22,3 +22,7 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get the information for the role - or a collection if you replace `role` with `collection`:
```
Comment (sbrl) [inline pages/common/ansible-galaxy.md:26]: Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Get the information for the role - or a collection if `role` is replaced with `collection`:
```

File: pages/common/ansible-galaxy.md [hunk 7]
```diff
@@ -22,3 +22,7 @@
...
+
+`ansible-galaxy role info {{username}}.{{role_name}}`
```
Comment (marchersimon) [inline pages/common/ansible-galaxy.md:28]: How about we rephrase this a bit and split it into two examples? Also adresses @sbrl's suggestion above
```suggestion
- Get information about a role:

`ansible-galaxy role info {{username}}.{{role_name}}`

- Get information about a collection:

`ansible-galaxy collection info {{username}}.{{collection_name}}`
```
(I'm not sure if the second one is correct tough)

Comment (papanito) [reply pages/common/ansible-galaxy.md]: Seems good

**Review by marchersimon — APPROVED** [review]
Thanks @papanito

File: pages/common/ansible-galaxy.md [hunk 8]
```diff
@@ -22,3 +22,11 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get information about a role:
```
Comment (navarroaxel) [inline pages/common/ansible-galaxy.md:26]: Is information about a specific user's role? or system role?

Comment (papanito) [reply pages/common/ansible-galaxy.md:26]: User role

Comment (sbrl) [reply pages/common/ansible-galaxy.md:26]: Perhaps this will read better?
```suggestion
- Get information about a user role:
```

Comment (navarroaxel) [reply pages/common/ansible-galaxy.md:26]: @papanito could you apply this change?

**Review by navarroaxel — APPROVED** [review]

**Comment by papanito** [comment]
Thanks
